Main.java

```diff
@@ -1,80 +1,122 @@
+            }
+        }
+        if (playerWinner == true) {
+            System.out.println();
+            System.out.println("\nTHIS IS THE END OF THE GAME. THANK YOU");
+            return; // return void - zwraca pustke I WTEDY WYCHODZI WCZESNIEJ Z METODY
+        } else {
+            putSignQuestions(player1);
+            checkLastMove(player1);
         }
-        putSignQuestions(player1);
-        checkLastMove(player1);
-        System.out.println("/nTHIS IS THE END OF THE GAME. THANK YOU");
+        System.out.println("\nTHIS IS THE END OF THE GAME. THANK YOU");
     }
 
-    static public void printTable(){
+    static public void printTable() {
+        System.out.println("\nTABLE DESIGN:\n---------");
         for (int i = 0; i < 3; i++) {
-            System.out.println(gameTable[i][0] + " " + gameTable[i][1] + " " + gameTable[i][2]);
+            System.out.println("| " + gameTable[i][0] + " " + gameTable[i][1] + " " + gameTable[i][2] + " |");
         }
+        System.out.println("---------");
     }
 
-    static public void enterCoordinates(Player player){  // TRZEBA ZROBIĆ OBSŁUGĘ WYJĄTKÓW - ARRAYINDEXOUTOFBOUND
-        System.out.println(player.getName() + " Give a column number where you want to put a sign");
-        //int
-        columnNumber = scanner.nextInt();
-        System.out.println(player.getName() + " Give a raw number where you want to put a sign");
+    static public void enterCoordinates(Player player) {  // TRZEBA ZROBIĆ OBSŁUGĘ WYJĄTKÓW - ARRAYINDEXOUTOFBOUND
+        System.out.println("\n" + player.getName() + ", Give a raw number where you want to put a sign");
         //int
         rawNumber = scanner.nextInt();
+        System.out.println("\n" + player.getName() + ", Give a column number where you want to put a sign");
+        //int
+        columnNumber = scanner.nextInt();
+    }
+
+    static public void markField(Player player) {
+        gameTable[rawNumber][columnNumber] = player.getSymbol();
     }
 
-    static public void markField(Player player){
-        gameTable[columnNumber][rawNumber] = player.getSymbol();
+    static public void markComputerField(Player player, int rawNumber, int columnNumber) {
+        gameTable[rawNumber][columnNumber] = player.getSymbol();
     }
 
-// CZY METODA STATYCZNA (CZYLI NIE POTRZEBUJĄCA KONSTRUKTORA DO WYWOŁANIA, TAK?) MOŻE BYĆ PRIVATE, DEFAULT?
+    static public void computerMove(Player player) {
+        generateNumber();
+        while (checkFreeField() == false)
+            generateNumber();
+        markField(player);
+    }
+
+    static public void generateNumber() {
+        Random generator = new Random();
+        rawNumber = generator.nextInt(3);
+        columnNumber = generator.nextInt(3);
+    }
+
+    // CZY METODA STATYCZNA (CZYLI NIE POTRZEBUJĄCA KONSTRUKTORA DO WYWOŁANIA, TAK?) MOŻE BYĆ PRIVATE, DEFAULT?
     static public void putSignQuestions(Player player) { // bo Main jest metodą statyczną!
         freeFieldVar = false;
```

Comment:
to make this code readable this variable should be local and checkIfFieldIsFree() should return true / false and it should be assigned to isFieldFree variable